test/promql/tests/data_plane/pipeline_deadlock_crashloop_test.yaml

```diff
@@ -0,0 +1,66 @@
+evaluation_interval: 1m
+
+rule_files:
+  - prometheus.pipeline_alerts.yaml
+
+tests:
+  # ----- Tekton Controller Rapid Restart Tests ----
+  - interval: 1m
+    input_series:
+
+      # see https://prometheus.io/docs/prometheus/latest/configuration/unit_testing_rules/ for explanations of the expanding notation used for the values
+      # tekton pipeline controller is experiencing controller successive restarts so it will be alerted, whereas we are currently not checking
+      # other pods in the same namespace that restart
+      - series: 'kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod="tekton-pipelines-controller-a", source_cluster="cluster01"}'
+        values: '1+1x780'
+      - series: 'kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod="tekton-pipelines-controller-a", source_cluster="cluster01"}'
+        values: '1+1x780'
```

Comment:
This input series is identical to the one above. Time series should be unique when comparing all labels so I think this should be removed.

Reply:
yep forgot how the `+` operator worked for a sec there   .... updating